.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,10 @@
+- id: jupytext
+  name: jupytext
+  description: Convert all notebooks into another format
+  language: python
+  entry: jupytext --add-untracked
+  pass_filenames: true
+  files: ipynb
```

Comment:
What about the other use case of generating ipynb from newly added Python files?

Reply:
This should be use the type: precommit value instead of the files. The former is much more robust and is the recommended way to make notebook pre-commit hooks.

Reply:
yep we already discussed the defaults in the issue, default behavior needs to be removed entirely. We do need a flag to ignore unused inputs instead of the current erroring.